v2/concordium/service.proto

```diff
@@ -253,4 +253,11 @@ service Queries {
   // This endpoint is only supported for protocol version 6 and onwards.
   // If the protocol does not support the endpoint then an  'IllegalArgument' error is returned.
   rpc GetBakersRewardPeriod(BlockHashInput) returns (stream BakerRewardPeriodInfo);
+
+  // For a non-genesis block, this returns the quorum certificate, a timeout
+  // certificate (if present) and epoch finalization entry (if present).
+  // If the node does not support ConcordiumBFT (protocol versions < 6), then
+  // the response will be a grpc error (invalid argument) as the 'BlockHashInput'
+  // does not point to a block produced via ConcordiumBFT.
```

Comment:
This does not explain the behaviour entirely.

Example 1:
Node version 6 supports protocol 6, but for example it is not enabled yet. WHat is the response?

Example 2:
Protocol 6 is in effect, but BlockHashInput points to a block at a different protocol. What is the response?

I suppose you meant to say that if the block identifier points to a block from an old protocol version then the response is invalidArgument.